protocol/messenger_response.go

```diff
@@ -49,6 +50,7 @@ type MessengerResponse struct {
 	WatchOnlyAccounts             []*accounts.Account
 	Keypairs                      []*accounts.Keypair
 	AccountsPositions             []*accounts.Account
+	TokenPreferences              []walletsettings.TokenPreferences
```

Comment:
Any reason why didn't you include `TokenPreferences` in isEmpty, Mrshal and Merge functions?